### ISSUE TITLE
Release Date

### ISSUE THREAD

**Issue by awsaf49** [shkarupa-alex/tfgcvit#1]
@shkarupa-alex Thank you for open-sourcing your project. I think this project is yet to be released as model weights' are currently being loaded from the `local` directory,
https://github.com/shkarupa-alex/tfgcvit/blob/122f1c95888824af1e45918d1b0b95a3b8ac1e97/tfgcvit/model.py#L149
* As the original repo is yet to be fixed and not sure when it'll be done I'm wondering if there is any plan to make a `pre-release`?
* I've tried running the codes directly but it ran into error, but with some minor changes it was able to run nicely.  
* I've also checked the prediction using the `converted_weights.py`, it works nicely. 
* Is there any plan to migrate from `keras` to `tf.keras`? Nowadays, it seems `keras` is getting obsolete.

**Comment by shkarupa-alex** [comment]
Hi
Model is ready at 99% except removing some debug code like that you find and final testing on ImageNet2.

But due to these issues https://github.com/NVlabs/GCVit/issues/10 , https://github.com/NVlabs/GCVit/issues/11 , https://github.com/NVlabs/GCVit/issues/13 original GCVit is a bag of trash. 
Until that issues are not fixed and new weights not released it make no sense to finish my port.

**Comment by awsaf49** [comment]
@shkarupa-alex You are right. But I tried inference with your ported wights, and even with these issues prediction seems to match the actual label. I am a bit confused about how a model with these flaws can result in better performance and correct prediction.

**Comment by shkarupa-alex** [comment]
Yes, but without trust...

**Comment by shkarupa-alex** [comment]
GCViT were finally fixed and retrained.
My port with all fixes, checks and new weights now published.
